test/formal/checked_int_mul_12.py

```diff
@@ -39,8 +39,8 @@
 			sol_overflow_check_2 = AND(SLT(X, 0), EQ(Y, min_value))
 			sol_overflow_check = Or(sol_overflow_check_1 != 0, sol_overflow_check_2 != 0)
 		else:
-			sol_overflow_check = (sol_overflow_check_1 != 0)
+			sol_overflow_check = sol_overflow_check_1 != 0
```

Comment:
What do you think about disabling this rule from our `pylintrc` instead? We're doing it pretty much only in places where we have comparisons and I think parentheses add readability in those cases. I'd rather keep this code as is.

Reply:
Yeah, both ways are fine for me. When you only have one condition it is not that difficult to read.

Reply:
It always throws me off when you have `=` and `==` in it so I always put a parenthesis on it. And I've seen others do it too. I actually think that conditions should always be parenthesized when you have `=` in there.

Reply:
haha, no problem. Make sense, let's disable it in our pylintrc then :)